src/together/resources/finetune.py

```diff
@@ -82,6 +83,11 @@ def createFinetuneRequest(
     if warmup_ratio > 1 or warmup_ratio < 0:
         raise ValueError("Warmup ratio should be between 0 and 1")
 
+    if train_on_inputs is None:
+        raise ValueError("train_on_inputs cannot be None")
```

Comment:
This is not allowed by the typing annotation itself, I believe we should not check for inputs that are not documented as permissible for the function